CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added the ability to change the aspect ratio of individual day-of-the-week items
 - Added support for self-sizing month headers
 - Added a new `setContent(_:animated:)` function, enabling developers to perform animated content updates
+- Added Swift format integration to enforce consistent code style
```

Comment:
Should the changelog mention that the package now requires Swift 5.8+?

Reply:
Oh yeah, good call